metal/resource_metal_vlan.go

```diff
@@ -40,10 +50,23 @@ func resourceMetalVlan() *schema.Resource {
 
 func resourceMetalVlanCreate(d *schema.ResourceData, meta interface{}) error {
 	c := meta.(*packngo.Client)
+
+	facRaw, facOk := d.GetOk("facility")
+	metroRaw, metroOk := d.GetOk("metro")
+
+	if !facOk && !metroOk {
+		return friendlyError(errors.New("one of facility and metro must be configured"))
+	}
+
 	createRequest := &packngo.VirtualNetworkCreateRequest{
 		ProjectID:   d.Get("project_id").(string),
 		Description: d.Get("description").(string),
-		Facility:    d.Get("facility").(string),
+	}
```

Comment:
Let's add the VXLAN request field (which will be getting renamed from VLAN to VXLAN soon/today) https://github.com/packethost/packngo/pull/257

Reply:
It can only be used with Metro.

Reply:
https://github.com/packethost/packngo/releases/tag/v0.10.0